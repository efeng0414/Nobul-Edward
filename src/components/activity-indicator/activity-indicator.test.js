import React from "react";
import renderer from "react-test-renderer";
import ActivityIndicator from "./index";

describe("ActivityIndicator", () => {
  const componentProps = {
    children: "activity-indicator",
    type: "loading",
    size: "large",
    spinning: false
  };
  const componentJSX = <ActivityIndicator {...componentProps} />;

  describe("component Snapshot", () => {
    const componentSnapshot = renderer.create(componentJSX).toJSON();
    it("should match existing snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
