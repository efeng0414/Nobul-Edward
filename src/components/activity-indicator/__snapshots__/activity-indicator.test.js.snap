// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityIndicator component Snapshot should match existing snapshot 1`] = `
<div
  className="ant-spin-nested-loading spinner-wrapper"
  style={null}
>
  <div
    className="ant-spin-container"
  >
    activity-indicator
  </div>
</div>
`;
